docker/py3-native/pyproject.toml

```diff
@@ -69,6 +69,9 @@ python-whois = "*"
 ipwhois = "*"
 edgegrid-python = "*"
 tldextract = "*"
+pydantic = "*"
```

Comment:
would lock to > version 2. Better to make sure in case something else gets added that it wont suddenly get switched